src/node/node.go

```diff
@@ -57,13 +57,13 @@ func NewNode(config config.SwanConfig) (*Node, error) {
 		joinRetryInterval: time.Second * 5,
 	}
 
-	err = os.MkdirAll(config.DataDir+"/"+nodeID, 0644)
+	err = os.MkdirAll(config.DataDir+nodeID, 0766)
 	if err != nil {
 		logrus.Errorf("os.MkdirAll got error: %s", err)
 		return nil, err
 	}
 
-	db, err := bolt.Open(config.DataDir+"/"+nodeID+"/swan.db", 0644, nil)
+	db, err := bolt.Open(config.DataDir+nodeID+"/swan.db", 0644, nil)
```

Comment:
in this way we should make sure the config.DataDir suffix is "/". but the duplicate "/" in there  it seem ok